public/Aryan Suri/Sidebar/sidebar.css

```diff
@@ -536,8 +506,30 @@ a.toggler.off {
 }
 
 @media print {
-  #custom_open, .custom_sidebar {
-    display:none;
+  #custom_open {
+    display: none;
+  }
+  #custom_sidebar {
+    display: none;
+  }
+  #sidebar1 {
+    display: none;
+  }
+  #sidebar2 {
+    display: none;
+  }
+  #sidebar3 {
+    display: none;
+  }
+  #sidebar4 {
+    display: none;
+  }
+  section.mt-content-container {
+    margin-left: 0px !important;
+    margin-right: 0px !important;
+  }
+  section.mt-content-container {
+    font-size: 1.1 rem !important;
+    text-align: justify !important;
```

Comment:
Are these `!important` essential to override another !important, or can you craft a more specific query that will be usable instead?

Reply:
these are important because there are in media print.